tests/test_utils/asserts.py

```diff
@@ -74,11 +77,16 @@ def assert_queries_count(expected_count, message_fmt=None):
         yield None
 
     count = sum(result.values())
-    if expected_count != count:
-        message_fmt = message_fmt or "The expected number of db queries is {expected_count}. " \
-                                     "The current number is {current_count}.\n\n" \
-                                     "Recorded query locations:"
-        message = message_fmt.format(current_count=count, expected_count=expected_count)
+    if count != expected_count:
+        message_fmt = (
+            message_fmt
+            or "The expected number of db queries is {expected_count}. "
+            "The current number is {current_count}.\n\n"
+            "Recorded query locations:"
```

Comment:
Could you revert the changes to this file please (since this is now just whitespace only in this file)